src/api/services/queries/data.ts

```diff
@@ -9,13 +9,12 @@ import {
 
 export function useListKeys(
   neighborhood: Network | undefined,
-  address: string = ""
+  address: string = "",
+  params: {}[]
```

Comment:
I'm surprised this didn't give you a warning. It looks like a required parameter (`params`) after an optional one (`address`).

What exactly is `params` in this case? Can we be more descriptive?